samples/bluetooth/shell_bt_nus/README.rst

```diff
@@ -17,31 +17,42 @@ Requirements
 
 * One of the following development boards:
 
-  * |nRF5340DK|
-  * |nRF52840DK|
-  * |nRF52DK|
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set1_start
+   :end-before: set1_end
 
 * A second nRF52 Development Kit board (PCA10040) for connecting with bt_nus_shell.py.
-  Alternatively, you can use :ref:`ble_console_readme` for connecting (Linux only).
+  Alternatively, you can use :ref:`ble_console_readme` for connecting, using Linux only.
 
 
 Building and running
 ********************
 
-This sample can be found under :file:`samples/bluetooth/shell_bt_nus` in the
-|NCS| folder structure.
+This sample can be found under :file:`samples/bluetooth/shell_bt_nus` in the |NCS| folder structure.
 
-See :ref:`gs_programming` for information about how to build and program the
-application.
+See :ref:`gs_programming` for information about how to build and program the application.
 
 Testing
 =======
 
 After programming the sample to your board, test it by connecting to it and sending some shell commands.
 
-.. include:: ../../../include/shell/shell_bt_nus.rst
-   :start-after: testing_start
-   :end-before: testing_end
+Perform the following steps to use the ``bt_nus_shell.py`` script:
+
+1. Compile the application ``shell_bt_nus`` and flash it to one of the Nordic Development Kit listed in the requirements section.
```

Comment:
This line is duplicating info from line 33.

Reply:
fixed